recipes/fenics-oasis/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set name = "fenics-oasis" %}
 {% set version = "2019.1" %}
-{% set python_min = "3.7" %}
+{% set python_min = "3.9" %}
```

Comment:
You only need to set this one if your min is higher than conda-forge's min. Otherwise, remove it and let the defaults be used.

```suggestion
```

Reply:
@ocefpaf I have removed `python_min` so conda-forge defaults are used. Thanks for the feedback!